build_web_compilers/lib/src/dev_compiler_builder.dart

```diff
@@ -114,56 +115,33 @@ Future<void> _createDevCompilerModule(Module module, BuildStep buildStep,
       '-o',
       jsOutputFile.path,
       debugMode ? '--source-map' : '--no-source-map',
+      for (var dep in transitiveDeps) _summaryArg(dep),
+      '--packages=${packagesFile.absolute.uri}',
+      '--module-name=${ddcModuleName(jsId)}',
+      '--multi-root-scheme=$multiRootScheme',
+      '--multi-root=.',
+      '--track-widget-creation',
+      '--inline-source-map',
+      if (useIncrementalCompiler) ...[
+        '--reuse-compiler-result',
+        '--use-incremental-compiler',
+      ],
+      // The urls to compile, using the package: path for files under lib and the
+      // full absolute path for other files.
+      ...module.sources.map((id) {
```

Comment:
nit: `for (var id in module.sources) _sourceArg(id)` (and extract out the logic), for consistency?

Reply:
Good call